.github/workflows/check-markdown-format.yaml

```diff
@@ -16,5 +11,5 @@ jobs:
         with:
           fetch-depth: 0
       - run: |
-          BRANCH_NAME=${{ github.event.repository.default_branch }}
+          export BRANCH_NAME=origin/${{ github.event.repository.default_branch }}
```

Comment:
Won't this always set BRANCH_NAME to master? Don't we want this to be the branch we are working on to compare differences with master?